.github/workflows/release.yml

```diff
@@ -0,0 +1,17 @@
+name: Releases
+
+on: 
+  push:
+    tags:
+    - '*'
```

Comment:
Should we only run on version-like tags for safety? e.g. `"*\\.*\\.*"` (not sure if that's the best way to encode it)

Reply:
I think GitHub only supports like '*.*.*'? Not sure if you need to escape the periods.

Reply:
I will look into it and add that.

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

I considered doing just numbers `[0-9]+`. However, that seems even more restrictive over what it currently is. But I can easily add that too if you want.

Reply:
Oh nice they must have added that, `[0-9]+.[0-9]+.[0-9]+*` seems sensible? with a trailing `*` for things like release candidates

Reply:
I don't have strong feelings it seems simple as-is too.

Reply:
I added the more restrictive pattern and tested it - works well now. 
Just need to keep this pattern in mind for if you ever change the release pattern. Shouldn't be a problem. But you never know where we will all be in the future